pkg/apis/core/v1alpha1/portforward_types.go

```diff
@@ -54,19 +54,25 @@ type PortForwardList struct {
 // PortForwardSpec defines the desired state of PortForward
 type PortForwardSpec struct {
 	// The name of the pod to port forward to/from. Required.
-	Pod string `json:"pod"`
+	PodName string `json:"pod_name"`
```

Comment:
```suggestion
	PodName string `json:"podName"`
```

FYI apiserver names are `camelCase` (standards doc still forthcoming)

Reply:
🤯  cool cool ty